### PR TITLE
Enable backtrace for enclave errors

### PR DESCRIPTION
**Pull request by podcastinator**
OE version 0.8 supports backtrace; this commit fixes LOG(FATAL) to capture a backtrace on an error, and write it out to a thread-local variable on the host via an ocall.
The ocall is required because thread-locals within enclaves are erased on an enclave exit; hence, retrieving the error message is no longer possible from the application.

### CODE REVIEW

File: include/enclave/crypto.h [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #include <mbedtls/x509_crt.h>
 #include <mbedtls/error.h>
 
-#include "mbedtls/gcm.h"
```
Comment (chester-leung) [inline include/enclave/crypto.h:19]: Was this needed before?

**Comment by chester-leung** [comment]
Looks good! Left one small question.

File: include/enclave/crypto.h (hunk 1 above)
Comment (podcastinator) [reply include/enclave/crypto.h:19]: Oh this was a duplicate include.

**Review by chester-leung — APPROVED** [review]

**Comment by chester-leung** [comment]
Thanks!

**Pull request merged**